src/api-fetch-setup.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import { fetchRequest } from 'react-native-gutenberg-bridge';
+
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+const fetchHandler = ( options ) => {
+	const responsePromise = fetchRequest( options.path );
+
+	const parseResponse = ( response ) => {
+		return response;
+	};
+
+	return responsePromise.then( parseResponse ).catch( ( error ) => {
```

Comment:
Do we need the `then`/`error` if they're just returning the passed value?

Reply:
I've updated `parseResponse` so it no longer just returns the passed value like it used to. I made this change because I have Android returning a json string because that's much simpler to do. If there's an issue with that I can certainly update it to return a json object like iOS.

With respect to the `catch` clause, I believe adding that does change the behavior so that now the resulting promise essentially "resolves" with the error (i.e., the resulting promise will pass the error in any subsequent `then` calls). Seems like this probably is _not_ desired behavior. What do you think @etoledom ?